### ISSUE TITLE
ComfyUI update broke LLM Party

### ISSUE THREAD

**Issue by IsItDanOrAi** [heshengtao/comfyui_LLM_party#82]
Updated ComfyUI and it broke LLM-Party. Anyone have any idea the cause?

Python 3.11.8
Win10
Pytorch 2.3.0+cu121
Torchvision 0.18.0+121
xformers 0.0.26.post1
numpy 1.26.4
pillow - n/a
OpenCV 4.10
transformers 4.41.1
diffusers 0.30.1
Cuda 12.1

**Comment by heshengtao** [comment]
Since you didn't give me more information, I can't judge your problem. You can update the party again. Recently, I removed a lot of dependencies that are easy to report errors.

**Comment by heshengtao** [comment]
I set this dependency to be unnecessary. If it is not installed correctly, only the LVM loader will fail, and it will have no effect on other nodes. The reason for the error is that this dependency is not adapted to cuda 12.4.

**Comment by IsItDanOrAi** [comment]
Just an update. I wanted to say thank you to both of you. 

Heshangtao. The node seems to install perfectly now. I uninstalled LLM-Party, and manually installed via Git Pull, as was mentioned, and it seems to work perfectly. The changes definitely helped, as I had tried this before. Thank you for the updates. Sorry if my original post didn't provide enough information to help locate the issue. But, I'm glad you were able to figure it out. 

NerdyRodent. Just wanted to say thank you for the clarification on the error, and for all your videos. Great work, and invaluable to the community.
